build-images.sh

```diff
@@ -4,6 +4,8 @@
 set -e
 
 MATTERMOST_VERSION=8.1.8
+alpine_version=3.19.1
+mattermost_ldap_version=2.1
```

Comment:
I'd use upper case variables since these are constants and we already use this convention.